libsolidity/codegen/CompilerContext.cpp

```diff
@@ -410,7 +410,7 @@ void CompilerContext::appendInlineAssembly(
 				util::errinfo_comment("Stack too deep (" + to_string(stackDiff) + "), try removing local variables.")
 			);
 		if (_context == yul::IdentifierContext::RValue)
-			_assembly.appendInstruction(dupInstruction(stackDiff));
+			_assembly.appendInstruction(dupInstruction(static_cast<size_t>(stackDiff)));
```

Comment:
`stackDiff` is already `size_t`.
```suggestion
			_assembly.appendInstruction(dupInstruction(stackDiff));
```